app/build.gradle

```
@@ -1,15 +1,16 @@
 plugins {
     id 'com.android.application'
 }
+apply plugin: 'kotlin-android'
```

you can change this as below, for consistency
```suggestion
plugins {
    id 'com.android.application'
    id 'kotlin-android'
}
```